src/App/Enumeration.h

```diff
@@ -148,13 +148,13 @@ namespace App
             /// Assignment operator
             Enumeration & operator=(const Enumeration &other);
 
-            /// true iff our string representation matches other's
+            /// true if our string representation matches others
```

Comment:
I've explained to you, at some length, that I think these comment edits are a waste of time.  So, sorry, but I'm not going to put further effort in this direction.